tools/c7n_azure/tests/templates/provision.sh

```diff
@@ -29,3 +29,11 @@ if [ $# -eq 0 ] || [[ "$@" =~ "containerservice" ]]; then
 else
   echo "Skipping $rgName"
 fi
+
+# Deploy Azure Policy Assignment
+if [ $# -eq 0 ] || [[ "$@" =~ "policyassignment" ]]; then
+  az policy assignment create --display-name cctestpolicy --name cctestpolicy --policy '06a78e20-9358-41c9-923c-fb736d382a4d'
```

Comment:
Where does `06a78e20-9358-41c9-923c-fb736d382a4d` come from? A comment may be helpful.